src/forms/StudyInfoForm/NewStudyForm.js

```diff
@@ -139,7 +135,7 @@ const NewStudyForm = ({
           inputObject.releaseDate = null;
         }
         if (newStudy) {
-          submitValue({input: inputObject, workflowType: workflowType});
+          submitValue({input: inputObject});
```

Comment:
The default workflows will have to be updated to be empty, too.
https://github.com/kids-first/kf-api-study-creator/blob/4a387f40e7e2f7f36f1472fe33e1e45d74689972/creator/settings/production.py#L325